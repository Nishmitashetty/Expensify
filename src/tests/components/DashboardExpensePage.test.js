import React from 'react';
import {shallow} from 'enzyme';
import DashboardExpensePage from '../../components/DashboardExpensePage';


test('should render Dashboard Expense Page correctly',() =>
{
    const wrapper = shallow(<DashboardExpensePage />);
    expect(wrapper).toMatchSnapshot();
});
