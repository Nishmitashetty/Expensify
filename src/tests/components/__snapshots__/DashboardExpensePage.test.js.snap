// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard Expense Page correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
